lib/events/events.go

```diff
@@ -111,22 +111,26 @@ func (t EventType) MarshalText() ([]byte, error) {
 const BufferSize = 64
 
 type Logger struct {
-	subs   []*Subscription
-	nextID int
-	mutex  sync.Mutex
+	subs         []*Subscription
+	nextGlobalID int
+	mutex        sync.Mutex
 }
 
 type Event struct {
-	ID   int         `json:"id"`
-	Time time.Time   `json:"time"`
-	Type EventType   `json:"type"`
-	Data interface{} `json:"data"`
+	// Per-subscription sequential event ID. Named "id" for backwards compatibility with the REST API
+	SubscriptionID int `json:"id"`
+	// Global ID of the event across all subscriptions
+	GlobalID int         `json:"global_id"`
```

Comment:
I'm happy to be convinced that we don't need to keep this. It isn't actually used anywhere (as far as I can tell), but I do like the concept of all copies of an Event scattered across multiple Subscriptions having a common, globally-unique ID...